dts/bindings/rtc/microcrystal,rv-8263-c8.yaml

```diff
@@ -0,0 +1,30 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2024 Daniel Kampert
+# Author: Daniel Kampert <DanielKampert@Kampis-Elektroecke.de>
+
+description: Micro Crystal RV-8263-C8 RTC
+
+compatible: "microcrystal,rv-8263-c8"
+
+include:
+  - name: rtc-device.yaml
+  - name: i2c-device.yaml
+
+properties:
+    int-gpios:
+      type: phandle-array
+
+    clkout:
+      type: int
+      default: 32768
+      enum:
+        - 32768
+        - 8192
+        - 1024
+        - 64
+        - 32
+        - 1
+        - 0
+      description:
+        Selection of the CLKOUT frequency. Set to 0 to deactivate the CLKOUT signal (CLKOUT pin is LOW).
+        The default setting is 32768 and corresponds to the reset value of the register field.
```

Comment:
This output really should not be enabled by default :) Enabling by default is "invisible" to users when looking at the node in the devicetree, so they will likely expect it to be disabled. Having it enabled by default also leads to the need for a "disabled" value (0)

Instead, write it like this:
```
   clkout:
      type: int
      enum:
        - 32768
        - 8192
        - 1024
        - 64
        - 32
        - 1
      description:
        Enable CLKOUT at given frequency. When disabled, CLKOUT pin is LOW.
```

Reply:
Then from the driver, you can use  DT_INST_PROP_OR(inst, clkout, 0),